testapi.pm

```diff
@@ -1473,6 +1479,23 @@ sub type_password {
     type_string $string, secret => 1, max_interval => ($args{max_interval} // 100), %args;
 }
 
+=head2 enter_cmd
+
+  enter_cmd($string [, max_interval => <num> ] [, wait_screen_changes => <num> ] [, wait_still_screen => <num> ] [, secret => 1 ]
+  [, timeout => <num>] [, similarity_level => <num>] );
+
+A convenience wrapper around C<type_string>, that adds a linefeed to execute a
+command within a command line prompt.
+
+You can pass the same optional parameters as for C<type_string> function.
+
+=cut
+
+sub enter_cmd {
+    my ($string, %args) = @_;
+    type_string $string, lf => 1, %args;
```

Comment:
I'd avoid creating an intermediate hash when the only goal is forwarding the arguments:
```suggestion
    type_string shift, lf => 1, @_;
```

Reply:
done